modules/microshift-downloading-container-images.adoc

```diff
@@ -43,23 +43,90 @@ $ IMAGE_LIST_FILE=~/microshift-container-refs.txt
 $ IMAGE_LOCAL_DIR=~/microshift-containers
 ----
 
-. Run the following script to download the container images to the `${IMAGE_LOCAL_DIR}` directory:
+. Run one of the following scripts that matches your use case:
+
+.. Run the following script to pull the images from ${IMAGE_LOCAL_DIR} to the mirror registry:
```

Comment:
This would be push instead of pull? Transfering from a local directory to a registry.